### ISSUE TITLE
Improve the time poll interface

### ISSUE THREAD

**Issue by jmallach** [loomio/loomio#9614]
The interface to add optional time slots in a time poll and the interface for choosing them once created the poll is not very useful.

When creating, it doesn't allow you to create time slots in one single day and clone that day for several other dates.

When choosing the options, the interface doesn't group the options by date. Although, at least, they are ordered. They are shown vertically too, which I'm not sure it's a better option than horizontally.

https://framadate.org/ does this slightly better, although not ideal. I think https://doodle.com/ handles this much better.

Compare:

![image](https://user-images.githubusercontent.com/929712/222422085-2e126505-cb9d-4b80-8fea-382553667f44.png)

vs

![image](https://user-images.githubusercontent.com/929712/222422138-85d80dcf-1f3e-4a06-be60-5e265fe1789e.png)

**Comment by robguthrie** [comment]
You're absolutely right. Currently I've just done what I can with the time I wanted to invest it the tool.

With that time, I've done a bunch of work on enabling date format preferences, timezone support, ical support. The poll form keeps the last value when you click 'add time slot', so that adding the time on another date is less work.

However, yes, adopting a calendar layout for the vote form would be a big improvement. Happy to collaborate on this with you.

**Comment by Hamsterarsch** [comment]
I also feel that the time poll GUI could be much improved. Personally, I think [lettucemeet](https://lettucemeet.com/) or it's open source rewrite [cabbagemeet](https://github.com/maxerenberg/cabbagemeet/tree/master) solve this problem well UX wise. Perhaps it's possible to get some meaningful integration with cabbagemeet so that it's less work? Alternatively, cabbagemeet might be a good base to argue about what to build exactly.
